google/logging/v2/logging_gapic.yaml

```diff
@@ -161,11 +166,14 @@ interfaces:
     timeout_millis: 30000
     field_name_patterns:
       parent: parent
+    resource_name_treatment: NONE
   - name: GetLogMetric
     flattening:
       groups:
       - parameters:
         - metric_name
+        parameter_resource_name_treatment:
```

Comment:
It's easier to be consistent this way - I think future maintainers are less likely to make a mistake if all method-level settings are NONE.